File: Algorithm/9093_WordReversal/9093_WordReversal.py
import re
import sys

# 정수랑 문자열 조심
T = int(input())

for _ in range(T):
    sentence = sys.stdin.readline().split()
    reverse = sentence[0][::-1]
    for word in sentence[1:]:
        reverse += (' ' + word[::-1])
    print(reverse)

File: Algorithm/9093_WordReversal/9093_WordReversal.cpp

#include <iostream>
#include <string>

using namespace std;

int main()
{
    int T = 0;
    cin >> T;
    for (int i = 0; i < T; i++)
    {
        string reverse = "";
        string word = "";
        cin >> word;
        if (word != '\n')
        {
            for (int x = 0; x < word.length(); x++)
            {
                reverse += word[x];
            }
            reverse += ' ';
        }
        else{
            cout << reverse << endl;
        } 
    }
    return 0;
}